tests/unit_tests/plugins/yaml/test_all_yaml_plugins.py

```diff
@@ -30,13 +30,19 @@ def yield_plugins():
         fpath = resources.files("geoips") / "plugins/yaml"
         plugin_files = fpath.rglob("*.yaml")
         for pf in plugin_files:
-            yield yaml.safe_load(open(pf, "r"))
+            yield pf
 
 
-@pytest.mark.parametrize("plugin", yield_plugins())
+def gen_label(val):
+    """Generate the yaml name."""
+    return val.name
+
+
+@pytest.mark.parametrize("plugin", yield_plugins(), ids=gen_label)
 def test_is_plugin_valid(plugin):
     """Test if plugin is valid."""
-    if plugin["interface"] == "products":
-        product_validator.validate(plugin)
+    rplugin = yaml.safe_load(open(plugin, "r"))
+    if rplugin["interface"] == "products":
+        product_validator.validate(rplugin)
     else:
-        validator.validate(plugin)
+        validator.validate(rplugin)
```

Comment:
Does this resolve @jsolbrig issue with test_all_yaml_plugins not returning informative output?

Reply:
It resolves that file's output, there's also test_sectors.py that I'm going to modify to have a more informative output.

Reply:
Great!! So this will course both the pytest arguments issue and the more informative output for yaml tests issue?